components/tabs/tabs.js

```diff
@@ -552,22 +560,9 @@ class Tabs extends LocalizeCoreElement(ArrowKeysMixin(RtlMixin(FocusVisiblePolyf
 		}
 	}
 
-	_handleFocusEnd(e) {
-		if (e.relatedTarget && e.relatedTarget.role !== 'tab') {
-			this._focusSelected();
-		} else {
-			const nextFocusable = getNextFocusable(e.target, false);
-			if (nextFocusable) nextFocusable.focus();
-		}
-	}
-
-	_handleFocusStart(e) {
-		if (e.relatedTarget && e.relatedTarget.role === 'tab') {
-			const previousFocusable = getPreviousFocusable(e.target, false);
-			if (previousFocusable) previousFocusable.focus();
-		} else {
-			this._focusSelected();
-		}
+	_handleFocusOut(e) {
+		if (!e.relatedTarget || e.relatedTarget.role === 'tab') return;
+		this._resetFocusables();
 	}
 
 	_handlePanelSelected(e) {
```

Comment:
It occurs to me that I missed this case. If the panel is selected, we also set its associated tab as selected. If that's the case, we also want to change which tab is focusable as well.